server/src/test/java/io/pravega/schemaregistry/storage/impl/group/GroupPravegaTest.java

```diff
@@ -0,0 +1,757 @@
+        pravegaKVGroups.getGroup(null, groupName).join().create(SerializationFormat.Custom,
+                ImmutableMap.of(),
+                Boolean.TRUE, Compatibility.backward()).join();
+        assertTrue(tableStore.getEntry(GROUPS, new NamespaceAndGroup(null, groupName).toBytes(),
+                GroupsValue::fromBytes).join().getRecord().getState().equals(GroupsValue.State.Active));
+    }
+
+    @Test
+    public void testGetCurrentEtag() {
+        GroupProperties groupProperties = GroupProperties.builder().allowMultipleTypes(Boolean.FALSE).properties(
+                ImmutableMap.<String, String>builder().build()).serializationFormat(
+                SerializationFormat.custom("custom1")).compatibility(
+                Compatibility.forward()).build();
+        pravegaKVGroups.addNewGroup(null, groupName, groupProperties).join();
+        pravegaKVGroups.getGroup(null, groupName).join().create(SerializationFormat.Custom,
+                ImmutableMap.of(),
+                Boolean.TRUE, Compatibility.backward()).join();
+        Etag etag = pravegaKVGroups.getGroup(null, groupName).join().getCurrentEtag().join();
+        assertTrue(tableStore.getEntry(GROUPS, new NamespaceAndGroup(null, groupName).toBytes(),
+                GroupsValue::fromBytes).join().getRecord().getState().equals(GroupsValue.State.Active));
+        GroupsValue gv = tableStore.getEntry(GROUPS, new NamespaceAndGroup(null, groupName).toBytes(),
+                GroupsValue::fromBytes).join().getRecord();
+        Version version = tableStore.getEntry("_schemaregistry" + String.format("/%s.#.metadata/0", gv.getId()),
+                new TableKeySerializer().toBytes(new TableRecords.Etag()),
+                x -> TableRecords.fromBytes(TableRecords.Etag.class, x, TableRecords.Etag.class)).join().getVersion();
+        assertEquals(etag.etag(), version);
+    }
+
+    @Test
+    public void testGetLatestSchemas() {
+        //null case
+        GroupProperties groupProperties = GroupProperties.builder().allowMultipleTypes(Boolean.FALSE).properties(
+                ImmutableMap.<String, String>builder().build()).serializationFormat(
+                SerializationFormat.custom("custom1")).compatibility(
+                Compatibility.forward()).build();
+        pravegaKVGroups.addNewGroup(null, groupName, groupProperties).join();
+        pravegaKVGroups.getGroup(null, groupName).join().create(SerializationFormat.Custom,
+                ImmutableMap.of(),
+                Boolean.TRUE, Compatibility.backward()).join();
+        List<SchemaWithVersion> schemaWithVersionList = pravegaKVGroups.getGroup(
+                null, groupName).join().getLatestSchemas().join();
+        assertTrue(schemaWithVersionList.isEmpty());
+        //non-null case
+        byte[] schemaData = new byte[0];
+        SchemaInfo schemaInfo = new SchemaInfo("anygroup", SerializationFormat.Custom, ByteBuffer.wrap(schemaData),
```

Comment:
here and everywhere.. assign "anygroup" to a variable and access it everywhere.. that way you ensure that no typos are present as same string is used everywhere.